compose/parallel.py

```diff
@@ -50,7 +50,10 @@ def parallel_execute_watch(events, writer, errors, results, msg, get_name):
     error_to_reraise = None
     for obj, result, exception in events:
         if exception is None:
-            writer.write(msg, get_name(obj), 'done', green)
+            if callable(getattr(obj, 'containers', None)) and not obj.containers():
```

Comment:
Sorry, one last thing - can you add a comment here to explain what problem this solves, as a reminder for our future selves? :)